library/src/main/assets/google_map.html

```diff
@@ -43,6 +47,14 @@
   AirMapView.onMapLoaded();
 }
 
+function setChinaMode() {
+  isChinaMode = true;
```

Comment:
instead of having this variable (which is only used in one place), could we set an `iconUrlPrefix` (which is defined as non-CN prefix in variable declaration) when this method is called, so we can get rid of the additional `getMarkerIcon` method?